rust/private/common.bzl

```diff
@@ -31,7 +31,7 @@ load(":providers.bzl", "CrateGroupInfo", "CrateInfo", "DepInfo", "DepVariantInfo
 #
 # Note: Code in `.github/workflows/crate_universe.yaml` looks for this line, if
 # you remove it or change its format, you will also need to update that code.
-DEFAULT_RUST_VERSION = "1.84.0"
+DEFAULT_RUST_VERSION = "1.84.1"
 
 DEFAULT_NIGHTLY_ISO_DATE = "2024-11-28"
```

Comment:
Nightly is still unchanged due to https://github.com/frewsxcv/rust-crates-index/issues/182. Only when this issue is resolved can `crate_universe` begin using newer versions of Rust.